.storybook/main.js

```diff
@@ -2,6 +2,10 @@ const path = require('path');
 const toPath = filePath => path.join(process.cwd(), filePath);
 
 module.exports = {
+  typescript: { reactDocgen: 'react-docgen' },
```

Comment:
I'm not using the doc tab - so a win for me too!

Reply:
Ah I wonder if this can be set only in development 🤔 

It can be kinda useful, but nothing you can't see in the code (if we don't add anything extra!)

![image](https://user-images.githubusercontent.com/35858975/134134325-d02e697f-ed36-4604-95a0-40bbc612f2bc.png)